text-serializer-gson-legacy-impl/src/main/java/net/kyori/adventure/text/serializer/gson/legacyimpl/NBTLegacyHoverEventSerializer.java

```diff
@@ -31,14 +31,17 @@
  * A legacy {@link HoverEvent} serializer.
  *
  * @since 4.3.0
+ * @deprecated For removal, use {@link net.kyori.adventure.text.serializer.json.legacyimpl.NBTLegacyHoverEventSerializer}
```

Comment:
For consistency with the rest of the API. This applies throughout.
```suggestion
 * @deprecated for removal since 4.9.0, use {@link net.kyori.adventure.text.serializer.json.legacyimpl.NBTLegacyHoverEventSerializer} instead.
```